packs/opsgenie/actions/add_heartbeat.yaml

```diff
@@ -0,0 +1,34 @@
+---
+description: "Add a Heartbeat to OpsGenie"
+enabled: true
+entry_point: "add_heartbeat.py"
+name: "add_heartbeat"
+pack: "opsgenie"
+
+runner_type: "python-script"
+
+parameters:
+  name:
+    type: "string"
+    description: "Name of the heartbeat."
+    required: true
+  interval:
+    type: "integer"
+    description: "Specifies how often a heartbeat message should be expected."
+    default: ~
```

Comment:
It sure does work and the README has an example of using this action to setup a heartbeat between 2 ST2 instances to check ChatOps is functional.